nflgame/live.py

```diff
@@ -430,6 +429,8 @@ def calc_week(instant):
         if  week > 4:
             week = 4
 
+    # Let's not work on non-integer weeks
+    week = math.trunc(week)
```

Comment:
previously weeks were able to be set as something like 4.123 because it was just decimal division.  This ensure that we are only ever working on integer weeks.